pkg/docker/index.html

```diff
@@ -336,6 +338,45 @@ <h1 translatable="yes">Can't connect to Docker</h1>
     </div>
   </div>
 
+  <!-- STORAGE -->
+
+  <div id="storage" class="container-fluid" hidden>
+    <ol class="breadcrumb">
+      <li><a translatable="yes">Containers</a></li>
+      <li class="active">Storage</li>
+    </ol>
+
+    <div id="storage-unsupported" hidden>
+      <div class="blank-slate-pf">
+        <h1 id="storage-unsupported-message"></h1>
+      </div>
+    </div>
+
+    <div id="storage-details">
+      <div class="col-md-4">
+        <h3 translatable="yes">Overview</h3>
+        <div id="storage-overview"></div>
+        <h3>
```

Comment:
You cannot nest block elements inside an `<h3>` (or any other heading tag).

If you want to fix this easily, just swap the open and close `<div>` for `<span>`. 

[Heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) can only contain [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content) — which, thankfully, includes `<span>`, `<button>`, and others.